makefiles/features.inc.mk

```diff
@@ -1,6 +1,5 @@
 # import list of provided features
 -include $(RIOTBOARD)/$(BOARD)/Makefile.features
```

Comment:
wondering: with the boards `Makefile.features` now including the `CPU`s `Makefile.features`, shouldn't we force the existence of a Makefile.features for every board (-> removing the minus sign in this line)? As there might be features provided by a CPU that are not looked at in case a board does not provide a `Makefile.features`...

Reply:
Will do!

Reply:
done.